esp/src/src-react/components/Menu.tsx

```diff
@@ -298,8 +298,6 @@ export const SubNavigation: React.FunctionComponent<SubNavigationProps> = ({
     React.useEffect(() => {
         hasLogAccess().then(response => {
             setLogsDisabled(!response);
-        }).catch(() => {
-            setLogsDisabled(true);
```

Comment:
I missed this in the review a while back I think. But this catch and the one removed in WorkunitDetails are unreachable aren't they? Because of the catch in hasLogAccess (around line 56/57 in ESPLog.ts)